frontend/src/components/kl-main.js

```diff
@@ -3,7 +3,7 @@ import klChar from './kl-char.vue';
 import i18n from '../plugins/i18n.js';
 export default {
     data: function () {
-        return {};
+        return { store: this.$store };
```

Comment:
I'd suggest `return { '$store': this.$store };` so that you don't have both `this.store` and `this.$store` which are the same.